### PR TITLE
fix deploy_che.sh ability to be launched from any place

### PR DESCRIPTION
**Pull request by riuvshin**
fix deploy_che.sh ability to be launched from any place
needed for: https://github.com/eclipse/che/issues/9158

### CODE REVIEW

**Review by sleshchenko — APPROVED** [review]

File: deploy/openshift/deploy_che.sh [hunk 1]
```diff
@@ -93,30 +93,30 @@ wait_until_che_is_available() {
 }
 
 # --------------
-# Print Che logo
+# Print Che logo
```
Comment (benoitf) [inline deploy/openshift/deploy_che.sh]: FYI there is a trailing space

File: deploy/openshift/deploy_che.sh [hunk 2]
```diff
@@ -358,15 +359,15 @@ if ! oc get project "${CHE_OPENSHIFT_PROJECT}" &> /dev/null; then
     echo "Project \"${CHE_OPENSHIFT_PROJECT}\" does not exist...trying to create it."
     DEPLOYMENT_TIMEOUT_SEC=120
     POLLING_INTERVAL_SEC=2
-    timeout_in=$((POLLING_INTERVAL_SEC+DEPLOYMENT_TIMEOUT_SEC))
+    timeout_in=$((POLLING_INTERVAL_SEC+DEPLOYMENT_TIMEOUT_SEC))
```
Comment (benoitf) [inline deploy/openshift/deploy_che.sh]: trailing spaces

File: deploy/openshift/deploy_che.sh [hunk 3]
```diff
@@ -358,15 +359,15 @@ if ! oc get project "${CHE_OPENSHIFT_PROJECT}" &> /dev/null; then
...
+    timeout_in=$((POLLING_INTERVAL_SEC+DEPLOYMENT_TIMEOUT_SEC))  
     while $WAIT_FOR_PROJECT_TO_DELETE
     do
     { # try
         timeout_in=$((timeout_in-POLLING_INTERVAL_SEC))
         if [ "$timeout_in" -le "0" ] ; then
             echo "[CHE] **ERROR**: Timeout of $DEPLOYMENT_TIMEOUT_SEC waiting for project \"${CHE_OPENSHIFT_PROJECT}\" to be deleted."
             exit 1
-        fi
+        fi
```
Comment (benoitf) [inline deploy/openshift/deploy_che.sh]: trailing spaces

**Review by ghost — APPROVED** [review]

File: deploy/openshift/deploy_che.sh (hunk 3 above)
Comment (riuvshin) [reply deploy/openshift/deploy_che.sh]: fixed

File: deploy/openshift/deploy_che.sh (hunk 2 above)
Comment (riuvshin) [reply deploy/openshift/deploy_che.sh]: fixed

File: deploy/openshift/deploy_che.sh (hunk 1 above)
Comment (riuvshin) [reply deploy/openshift/deploy_che.sh]: fixed

**Review by garagatyi — APPROVED** [review]

**Pull request merged**